include/cadet/ModelSystem.hpp

```diff
@@ -165,12 +164,9 @@ class CADET_API IModelSystem
 
 
 	/**
-	 * @brief Return the index and length of state vector relavant to each unit operation
-	 * @details This function returns starting index and stride of the last_state_vector
-	 *			related to each unit operation. This will faciliate the user to read
-	 *			the state vector in a more efficient manner.
-	 */
-	virtual const unsigned int* getModelSlice() const CADET_NOEXCEPT=0;
+	* @brief Return the offset indices corresponding to the last state of each unit operation
+	*/
+	virtual unsigned int const* getModelStateOffsets() const CADET_NOEXCEPT=0;
```

Comment:
Much better, but the documentation is missing a `@return` description.
Also, it is the offset to the local slice of each model in the global state vector.

Please use spaces at `... CADET_NOEXCEPT = 0;`.